host/calls.c

```diff
@@ -114,14 +115,21 @@ oe_result_t oe_call_enclave_function_by_table_id(
 
 oe_result_t oe_call_enclave_function(
     oe_enclave_t* enclave,
-    uint32_t function_id,
+    uint64_t* global_id,
+    const char* name,
     const void* input_buffer,
     size_t input_buffer_size,
     void* output_buffer,
     size_t output_buffer_size,
     size_t* output_bytes_written)
 {
-    return oe_call_enclave_function_by_table_id(
+    oe_result_t result = OE_UNEXPECTED;
+    uint64_t function_id = OE_UINT64_MAX;
+
+    OE_CHECK(
```

Comment:
I'd like to carry the following comments.

```suggestion
        /*
         * Look up the function id from the per-enclave table based on the
         * global id. The global id is defined as a static variable in the
         * oeedger8r-generated code. The function initializes the global id in
         * the first invocation and uses the cached global id for the subsequent
         * invocations.
         */
         OE_CHECK(
```